crypto/provider_child.c

```diff
@@ -135,8 +135,10 @@ static int provider_create_child_cb(const OSSL_CORE_HANDLE *prov, void *cbdata)
                                        1)) == NULL)
             goto err;
 
-        if (!ossl_provider_activate(cprov, 0, 0))
+        if (!ossl_provider_activate(cprov, 0, 0)){
```

Comment:
Please add a space before `{`

Reply:
Sorry about the formatting problem! I've updated it.